Kulyk_Lab/app/src/main/java/com/nazar/kulyk_lab/models/artObjects/ArtObject.java

```diff
@@ -74,4 +76,9 @@ public String getTitle() {
     public String getLongTitle() {
         return longTitle;
     }
+
+    @Override
+    public boolean equals(Object obj) {
+        return super.equals(obj);
```

Comment:
I don't think you actually need this method, since you call parent implementation anyway